blobmodel/model.py

```diff
@@ -57,6 +61,15 @@ def __init__(
             self.y = np.arange(0, self.Ly, self.Ly / self.Ny)
         self.t: NDArray[Any, Float[64]] = np.arange(0, self.T, self.dt)
 
+        self.blob_factory = blob_factory
+        self.__blobs = self.blob_factory.sample_blobs(
+            Ly=self.Ly,
+            T=self.T,
+            num_blobs=self.num_blobs,
+            blob_shape=self.blob_shape,
+            t_drain=self.t_drain,
+        )
```

Comment:
This might be personal preference, but I think this line should be in the integrate method. That way we separate the model variables, which are initialized in the constructor, and a particular realization of the process which happens in the integrate method. In principle a single model instance can lead to several realizations of the model. What do you think?

Reply:
yeah, that would make the code easier to understand